root/static/scripts/edit/URLCleanup.js

```diff
@@ -501,7 +548,11 @@ const CLEANUPS = {
       new RegExp("^(https?://)?(www\\.)?avexnet\\.jp/id/[a-z0-9]{5}/discography/product/[A-Z0-9]{4}-[0-9]{5}\\.html$", "i"),
       new RegExp("^(https?://)?(www\\.)?kingrecords\\.co\\.jp/cs/g/g[A-Z]{4}-[0-9]+/$", "i")
     ],
-    type: LINK_TYPES.discographyentry
+    type: LINK_TYPES.discographyentry,
+    validate: function (url, id) {
+      // avoid wikipedia being added as release-level discography entry
+      return !/^(https?:\/\/)?([^.\/]+\.)?wikipedia\.org\//.test(url);
```

Comment:
We have a problem here: For a Wikipedia URL, this function isn’t run because the URL doesn’t match the `match` expressions. I could obviously add a regex for WP there, but that would lead to “discographyentry” being auto-selected when a WP URL is pasted. This would likely confuse the user even more than they already are, trying to make a release–WP link.